### ISSUE TITLE
what is the difference between DSOD300_pascal and DSOD300_pascal++?

### ISSUE THREAD

**Issue by wellingtonyl** [szq0214/DSOD#13]
@szq0214 
what is the difference between DSOD300_pascal and DSOD300_pascal++?

**Comment by szq0214** [comment]
Hi @wellingtonyl, DSOD300_pascal uses training data of VOC 07 trainval + 12 trainval and tests on VOC 07 test set,  DSOD300_pascal++ uses more training data (VOC 07 trainval + 07 test + 12 trainval) than DSOD300_pascal and tests on VOC 12 test set.

**Comment by wellingtonyl** [comment]
got it, thanks
